webstruct/sequence_encoding.py

```diff
@@ -36,7 +44,7 @@ class IobEncoder(object):
 
     Group results to entities::
 
-        >>> iob_encoder.group(iob_encoder.encode(input_tokens))
+        >>> iob_encoder.group([p for p in IobEncoder.from_indicies(iob_encoder.encode(input_tokens), input_tokens)])
```

Comment:
`IobEncoder.from_indicies(iob_encoder.encode(input_tokens), input_tokens)` is repeated in all examples. I wonder if we should keep `encode` method backwards compatible, and introduce another method, which uses indices; `encode` implementation would be `IobEncoder.from_indicies(iob_encoder.encode(input_tokens), input_tokens)`.

Reply:
This pattern is repeated in test cases only, we can define a function in tests code.